corehq/apps/hqcase/api/get_list.py

```diff
@@ -119,7 +135,12 @@ def _get_query(domain, params):
              .sort("@indexed_on")
              .sort("_id", reset_sort=False))
     for key, val in params.items():
-        query = query.filter(_get_filter(domain, key, val))
+        if isinstance(val, list):
+            # e.g. key='owner_id', val=['abc123', 'def456']
+            filter_list = [_get_filter(domain, key, v) for v in val]
+            query = query.filter(filters.OR(*filter_list))
+        else:
+            query = query.filter(_get_filter(domain, key, val))
```

Comment:
If the new format of `params` still needs to be conditionally processed here, why is the `unpack_query_dict()` function needed? The original `QueryDict` has all the necessary information without unpacking it as an intermediate step. It seems like all the "how to turn a `QueryDict` into an actual filter" logic should live right here, rather than being partially abstracted in an intermediate function.

```suggestion
        if len(val) == 1:
            query = query.filter(_get_filter(domain, key, val[0]))
        else:
            # e.g. key='owner_id', val=['abc123', 'def456']
            filter_list = [_get_filter(domain, key, v) for v in val]
            query = query.filter(filters.OR(*filter_list))
```